ark/phenotyping/run_cell_som.R

```diff
@@ -1,10 +1,11 @@
 # Assigns cluster labels to cell data using a trained SOM weights matrix
 
-# Usage: Rscript run_cell_som.R {clusterCountsPath} {cellWeightsPath} {cellClusterPath}
+# Usage: Rscript run_cell_som.R {clusterCountsPath} {clusterCountsNormPath} {cellWeightsPath} {cellClusterNormPath}
```

Comment:
Can remove {clusterCountsPath} here, since {ClusterCountsNormPath} was added. Only reading in 3 arguments.